### DISCUSSION TITLE
LCD/OLED Screen Implementation for your DUCO stats

### DISCUSSION THREAD

**Discussion by Darksplat** [revoxhere/duino-coin#1197] [Ideas]
Hi guys is there a way I can display my web wallet on my RPI3 on an oled screen(SSD1351 chipset RGB 128x128) or LCD or OLED just the common ones I think the lcdgfx library could help. Instead of having to login on a computer all the time? This way I can just do it from a glance? I just want how much DUCO I have what it is approx in USD and how many miners are connected. Is that possible?

**Comment by codeoffun2** [comment]
If you’re going to make a suggestion, don’t make it sound like a question.

**Comment by nerudesu** [comment]
Of course, you can! Start reading the [API](https://github.com/revoxhere/duino-coin/tree/useful-tools) and make a function that called every 10s to fetch your account balance. I've done a pretty similar thing on a 16x2 LCD Display.

**Reply by Darksplat** [reply]
Nice but I am not a programmer!! I need someone to do it for us :)

**Comment by tbillion** [comment]
https://github.com/tbillion/ducostatsbasic

not oled but epaper, much more pretty and elegant ...

https://www.reddit.com/r/Duinocoin/comments/tbam59/epaper_statistic_display/?utm_source=share&utm_medium=web2x&context=3

written in python. 

I am a programmer, and YW :)
